Analysis/src/QwMollerADC_Channel.cc

```diff
@@ -1044,12 +1025,6 @@ void  QwMollerADC_Channel::FillNTupleVector(std::vector<Double_t>& values) const
       return;
     }
 
-    // For moments data (stat prefix), delegate to TTree behavior for exact match
-    if (fDataToSave == kMoments) {
-      // Use the existing TTree fill logic to ensure exact matching
-      FillTreeVector(values);
-      return;
-    }
```

Comment:
Note this change now required to FillNTupleVector since falling through to FillTreeVector is not correct anymore.